openvbi/adaptors/__init__.py

```diff
@@ -22,3 +22,19 @@
 # WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 # FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE
 # OR OTHER DEALINGS IN THE SOFTWARE.
+
+from typing import Protocol
+from pathlib import Path
+from openvbi.core.observations import Dataset
+
+class Loader(Protocol):
+    def suffix(self) -> str:
+        pass
+    def load(self, filename: str) -> Dataset:
```

Comment:
type should be `str | Path` to match the `filename` parameter of `Writer.write()`.